.github/workflows/docker-publish.yml

```
@@ -89,10 +99,35 @@
+            GIT_COMMIT=${{ needs.preBuild.outputs.commit }}
+            VERSION=${{ needs.preBuild.outputs.tag }}
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  multiarch:
+    needs: [ build, preBuild ]
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    steps:
+        - name: Log into registry ${{ env.REGISTRY }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)